runtime/src/bank.rs

```diff
@@ -7231,28 +7231,6 @@ impl Bank {
         self.update_accounts_hash_with_index_option(true, false, false)
     }
 
-    /// When the validator starts up, there is an initial hash calculation verification of the snapshot.
-    /// Then, in some conditions, there are additional slots replayed and a second hash calculation verification occurs.
-    /// This function is called when all initial hash calculations have been requested or completed.
-    pub fn initial_blockstore_processing_completed(&self) {
-        self.rc
-            .accounts
-            .accounts_db
-            .initial_blockstore_processing_complete
-            .store(true, Release);
-    }
-
-    /// until this occurs, no additional accounts hash calculations can be started
-    /// There will be 0..=2 of these requests.
-    pub fn is_initial_blockstore_processing_complete(&self) -> bool {
-        self.rc
-            .accounts
-            .accounts_db
-            .initial_blockstore_processing_complete
-            .load(Acquire)
-            && self.is_startup_verification_complete()
```

Comment:
`is_startup_verification_complete()`
has to still be called from whoever was calling `is_initial_blockstore_processing_complete`